### DISCUSSION TITLE
请问fe启动提示invalid host port

### DISCUSSION THREAD

**Discussion by gisIdea** [apache/doris#37321] [A - General / Q&A]
win10 64位系统，内网环境，vmware虚拟机里，运行ubuntu22.04系统，java1.8 gcc是10的版本，python是3的版本，，，mysql是5.7的版本，，然后用的是doris2.0.0的tar包，直接解压，进入fe.config配置好后，启动fe后，jps那个DorisFE显示在，可是8030那个页面打不开，日志里报错无效的host port ，，，请问大佬，这个错怎么处理，![mmexport1720139070030.jpg](https://github.com/apache/doris/assets/2839548/0d597aab-f48b-431f-8e95-3b3e8cce9b2a)

**Comment by ixzc** [comment]
中文问题在ask.selectdb.com提帖子
